bundle/Core/Persistence/Legacy/Content/FieldValue/Converter/Converter.php

```diff
@@ -62,7 +62,7 @@ public function toFieldDefinition(StorageFieldDefinition $storageDef, FieldDefin
                     'FileSizeValidator' => array(
                         'maxFileSize' => (0 !== $storageDef->dataInt1
                             ? $storageDef->dataInt1
-                            : false),
+                            : null),
```

Comment:
Why this change? Could be a breaking change.

Reply:
Hey @emodric ,

This is needed by the FieldMapper. If there is no maxFileSize stored for the field type yet it will be auto-assigned "false" which is not a numeric value, thus leading to an exception when trying to edit the content type in the backend.

![image](https://user-images.githubusercontent.com/8490373/49283706-bd1d0b80-f492-11e8-88df-bd3658af46f9.png)

Reply:
Thanks for the explanation. @MarioBlazek can you check if this change causes any regressions?

Reply:
Okay, it seems that this could indeed cause regressions:

https://github.com/netgen/NetgenEnhancedBinaryFileBundle/blob/master/bundle/Form/FieldTypeHandler/EnhancedFile.php#L71-L76.

You can change both those checks to add a check for null too. That should be enough I think.

Reply:
Do i? If someone updates the bundle it will both update `Netgen\Bundle\EnhancedBinaryFileBundle\Form\FieldTypeHandler\EnhancedFile` and `Netgen\Bundle\EnhancedBinaryFileBundle\Core\Persistence\Legacy\Content\FieldValue\Converter\Converter` so there won't be an inconsistency.

the values "false" or "null" (depending on installed version) will only come from the `Converter` class, never from database. Or is there an edge case i don't see?

Reply:
Yes, you are right, it doesn't come from the database, so this is okay!